docs/benchmarks.en.md

```
@@ -1,220 +1,243 @@
 title: "Performance Benchmarks"
 tags: [performance, benchmarks]
-audience: [platform-engineer, sre]
+audience: [platform-engineer, sre, decision-maker]
 version: v2.9.0
 lang: en
 ---
 # Performance Benchmarks
 
-> **Language / 語言：** [中文](./benchmarks.md) | **English (Current)**
+> **Language / 語言：** | [中文](./benchmarks.md) | **English (current)**
 
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)